kernel/packages/unity-interface/BrowserInterface.ts

```diff
@@ -377,6 +378,45 @@ export class BrowserInterface {
     globalThis.globalStore.dispatch(updateFriendship(action, userId.toLowerCase(), false))
   }
 
+  public SearchENSOwner(data: { name: string; maxResults: number }) {
+    let profilesPromise
+
+    if (/^0x[a-fA-F0-9]{40}$/.test(data.name)) {
```

Comment:
Could you make this test into a different function with a name like `isStringEthAddress` or similar. So that it is clearer what you are testing

Reply:
`import { isAddress } from 'eth-connect'`

Reply:
done. thx menduz!